reconcile/gql_definitions/common/app_interface_settings.py

```diff
@@ -0,0 +1,244 @@
+"""
+Generated by qenerate plugin=pydantic_v1. DO NOT MODIFY MANUALLY!
+"""
+from typing import (  # noqa: F401 # pylint: disable=W0611
+    Any,
+    Callable,
+    Optional,
+    Union,
+)
+
+from pydantic import (  # noqa: F401 # pylint: disable=W0611
+    BaseModel,
+    Extra,
+    Field,
+    Json,
+)
+
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
```

Comment:
Duplicated `VaultSecret` imports are a bug in https://github.com/app-sre/qenerate/. I will address later

Reply:
Build is failing because of this